packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/Actions.tsx

```diff
@@ -292,195 +299,146 @@ const BulkActionModal = ({
-//         toggleNotification({
-//           type: 'warning',
-//           message: formatAPIError(error),
-//         });
-//       },
-//     }
-//   );
-
-//   const handleConfirmUnpublishAllData = async () => {
-//     await bulkUnpublishMutation.mutateAsync({ ids });
-//   };
-
-//   const showUnpublishButton =
-//     hasPublishPermission && selectedEntriesObjects.some((entry) => entry.publishedAt);
-
-//   if (!showUnpublishButton) return null;
-
-//   return {
-//     actionType: 'unpublish',
-//     variant: 'tertiary',
-//     label: formatMessage({ id: 'app.utils.unpublish', defaultMessage: 'Unpublish' }),
-//     dialog: {
-//       type: 'dialog',
-//       title: formatMessage({
-//         id: 'app.components.ConfirmDialog.title',
-//         defaultMessage: 'Confirmation',
-//       }),
-//       content: (
-//         <Flex direction="column" alignItems="stretch" gap={2}>
-//           <Typography id="confirm-description" textAlign="center">
-//             {formatMessage({
-//               id: 'popUpWarning.bodyMessage.contentType.unpublish.all',
-//               defaultMessage: 'Are you sure you want to unpublish these entries?',
-//             })}
-//           </Typography>
-//           {hasI18nEnabled && (
-//             <Box textAlign="center" padding={3}>
-//               <Typography textColor="danger500">
-//                 {formatMessage(
-//                   {
-//                     id: getTranslation('Settings.list.actions.unpublishAdditionalInfos'),
-//                     defaultMessage:
-//                       'This will unpublish the active locale versions <em>(from Internationalization)</em>',
-//                   },
-//                   {
-//                     em: Emphasis,
-//                   }
-//                 )}
-//               </Typography>
-//             </Box>
-//           )}
-//         </Flex>
-//       ),
-//       confirmButton: formatMessage({
-//         id: 'app.utils.unpublish',
-//         defaultMessage: 'Unpublish',
-//       }),
-//       onConfirm: handleConfirmUnpublishAllData,
-//     },
-//   };
-// };
+const DeleteAction: BulkActionComponent = ({ documents, model }) => {
+  const { formatMessage } = useIntl();
+  const { schema: contentType } = useDoc();
+  const { selectRow } = useTable('deleteAction', (state) => state);
+  const hasI18nEnabled = Boolean(contentType?.pluginOptions?.i18n);
+  const [{ query }] = useQueryParams<{ plugins?: { i18n?: { locale?: string } } }>();
+  const currentLocale = query.plugins?.i18n?.locale || 'en';
+  const hasDeletePermission = useDocumentRBAC('deleteAction', (state) => state.canDelete);
+  const { deleteMany: bulkDeleteAction } = useDocumentActions();
+  const documentIds = documents.map(({ documentId }) => documentId);
+
+  const handleConfirmBulkDelete = async () => {
+    const res = await bulkDeleteAction({
+      documentIds,
+      model,
+      params: {
+        locale: currentLocale,
+      },
```

Comment:
Pass all the params that exist, you can see an example here:

https://github.com/strapi/strapi/blob/94f313a12ead5a0e467b870efd12f7e2a9d30531/packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx#L507